src/app/pages/storage/volumes/volumes-list/volumes-list.component.html

```diff
@@ -42,7 +42,7 @@
 
       <div fxLayoutAlign="end start" class="icon-row hidden">
         <div class="global_volume_based_action_menu pool-icon" title="Pool Operations" >
-          <app-entity-table-actions [icon_name]="'settings'" [entity]="actionComponent" [row]="row" [action]="'settings'" [groups]="true"></app-entity-table-actions>
+          <app-entity-table-actions style="margin-top:-4px;" icon_name="mdi-database-cog" [entity]="actionComponent" [row]="row" [action]="'settings'" [groups]="true"></app-entity-table-actions>
```

Comment:
seems `style="margin-top:-4px;"` has no effect on `app-entity-table-actions` since it is inline element